common/culture/cultures/wc_chelonian.txt

```diff
@@ -1,11 +1,16 @@
 ﻿tortollan = {
 	color = rgb { 60 55 35 }
 	
-	ethos = ethos_communal
+	ethos = ethos_egalitarian
```

Comment:
I can see that too, since they make the accumulation of knowledge and stories a lifelong pursuit. The reason I selected Communal is because of the gameplay effects. The positive cultural relations modifier fits their theme of travelling wide and far and finding accomodation with varying other cultures.

![image](https://user-images.githubusercontent.com/77478662/174688878-0571d7f9-3de6-4d7a-8e4d-7bf49a4e6b71.png)

Reply:
Sounds good!